python/ray/serve/utils.py

```diff
@@ -342,3 +342,37 @@ def get_node_id_for_actor(actor_handle):
     """Given an actor handle, return the node id it's placed on."""
 
     return ray.actors()[actor_handle._actor_id.hex()]["Address"]["NodeID"]
+
+
+def import_class(full_path: str):
```

Comment:
This is neat!  Maybe it could be useful elsewhere in Ray.  I almost feel like this should be a built-in function in Python, but maybe there are good reasons for it not to be.

Reply:
This is basically recreating:
`from module.submodule import Class` just using the lower-level API, so it sort of is a built-in function :)